source/Calamari.Shared/Integration/Packages/Java/JarTool.cs

```diff
@@ -35,27 +35,20 @@ public JarTool(ICommandLineRunner commandLineRunner, ICommandOutput commandOutpu
 
         public void CreateJar(string contentsDirectory, string targetJarPath)
         {
-            try
-            {
-                var manifestPath = Path.Combine(contentsDirectory, "META-INF", "MANIFEST.MF");
-                var args = File.Exists(manifestPath)
-                    ? $"-cp \"{toolsPath}\" sun.tools.jar.Main cvmf \"{manifestPath}\" \"{targetJarPath}\" -C \"{contentsDirectory}\" ."
-                    : $"-cp \"{toolsPath}\" sun.tools.jar.Main cvf \"{targetJarPath}\" -C \"{contentsDirectory}\" .";
-
-                var createJarCommand = new CommandLineInvocation(JavaRuntime.CmdPath, args, contentsDirectory);
-                Log.Verbose($"Invoking '{createJarCommand}' to create '{targetJarPath}'");
+            var manifestPath = Path.Combine(contentsDirectory, "META-INF", "MANIFEST.MF");
+            var args = File.Exists(manifestPath)
+                ? $"-cp \"{toolsPath}\" sun.tools.jar.Main cvmf \"{manifestPath}\" \"{targetJarPath}\" -C \"{contentsDirectory}\" ."
+                : $"-cp \"{toolsPath}\" sun.tools.jar.Main cvf \"{targetJarPath}\" -C \"{contentsDirectory}\" .";
 
-                /*
-                     All extraction messages should be verbose
-                 */
-                commandOutput.WriteInfo("##octopus[stdout-verbose]");
```

Comment:
At some point this stopped working as all the command output was piped through `Log.Info`